common/presenters/move-to-card-component.js

```diff
@@ -35,7 +34,7 @@ module.exports = function moveToCardComponent({ id, reference, person }) {
   return {
     href: `/move/${id}`,
     title: {
-      text: personService.getFullname(person).toUpperCase(),
+      text: person.fullname ? person.fullname.toUpperCase() : '',
```

Comment:
are we going to be in a situation where `person.fullname` doesn't exist?

Reply:
In theory, not anymore that is has been added to the Move service.

We may have to transform it if we extend the Person service to `GET` people.

My ideal place for this would have been in some transformation layer in the model but the devour library doesn't support that.